spec/features/register_spec.rb

```diff
@@ -0,0 +1,55 @@
+# spec/basic_spec.rb
+
+require 'page'
+
+
+RSpec.feature "Verify basic query search results" do
+
+  before do
+    @page = Page.new
+    @page.visit 'https://google.com'
+  end
+
+  feature "Upcase input" do
+
+    # not sure if curly bracket syntax is acceptable
+    before { @page.enter_keyword 'Monster' }
+
+    feature "First result" do
+      scenario "Title contains inputted keyword Upcase" do
+        expect(@page.first_result_title).to include('Monster')
+      end
+
+      scenario "URL contains inputted keyword Downcase" do
+        expect(@page.first_result_url).to include('monster')
```

Comment:
the thing is that url register is always downcased. knowing that we always expect it that way(downcase)